src/metaswitch/clearwater/config_manager/etcd_synchronizer.py

```diff
@@ -64,7 +64,7 @@ def main(self):
             if self._terminate_flag:
                 break
 
-            if len(value) > 0:
+            if value and len(value) > 0:
```

Comment:
This can just be `if value` - `""` (as well as `[]` and `{}`) are boolean false in Python.